README.md

```diff
@@ -2098,14 +2115,7 @@ set :protection, :except => [:path_traversal, :session_hijacking]
 ```
 
 By default, Sinatra will only set up session based protection if `:sessions`
-has been enabled. Sometimes you want to set up sessions on your own, though. In
-that case you can get it to set up session based protections by passing the
-`:session` option:
-
-```ruby
-use Rack::Session::Pool
-set :protection, :session => true
-```
+have been enabled. See 'Using Sessions'.
```

Comment:
Well my point was, this example was in the docs so it should have worked, unless something broke it and was missed because there wasn't a proper test covering it. :(